tests/bluetooth/bsim_bt/bsim_test_audio/src/broadcast_sink_test.c

```diff
@@ -19,6 +20,9 @@ CREATE_FLAG(pa_sync_lost);
 
 static struct bt_audio_broadcast_sink *g_sink;
 
+/* Mandatory support preset by both source and sink */
+static struct bt_audio_lc3_preset preset_16_2_1 = BT_AUDIO_LC3_BROADCAST_PRESET_16_2_1;
```

Comment:
In general(all sample apps) I think this variable name should be reduced to `preset` or `selected_preset` such that if I want to try for example 48 khz I simply select another preset rather than having to rename the variable all the places it is used :-)

Reply:
This is a test case though, but I understand what you mean. For this test case it can also easily just be called `preset`. Updated